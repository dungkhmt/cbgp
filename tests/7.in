30 30
30 50
17 10
22 0
5 0
25 13
8 29
16 22
19 0
10 6
15 0
20 3
27 7
11 15
10 12
1 3
16 2
22 3
4 5
19 24
17 1
26 2
17 27
23 0
19 25
22 8
29 14
19 9
21 22
16 18
28 0
12 9
5 2
19 4
13 0
13 4
22 1
23 2
24 2
19 13
19 6
23 15
14 8
22 18
26 16
21 8
19 2
28 19
12 1
16 15
12 14
22 20
1
15 15 35 5 18 5 34 6 14 7 16 7 46 0 43 2 35 2 15 1 45 1 22 7 40 1 12 9 37 2 21 6 24 3 38 8 23 2 3 0 33 7 34 3 28 2 19 6 15 6 12 8 18 4 41 2 0 0 36 3 