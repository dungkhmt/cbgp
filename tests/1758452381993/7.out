(17, 10), (22, 0), (5, 0), (25, 13), (8, 29), (16, 22), (19, 0), (10, 6), (15, 0), (20, 3), (27, 7), (11, 15), (10, 12), (1, 3), (16, 2), (22, 3), (4, 5), (19, 24), (17, 1), (26, 2), (17, 27), (23, 0), (19, 25), (22, 8), (29, 14), (19, 9), (21, 22), (16, 18), (28, 0), (12, 9), (5, 2), (19, 4), (13, 0), (13, 4), (22, 1), (23, 2), (24, 2), (19, 13), (19, 6), (23, 15), (14, 8), (22, 18), (26, 16), (21, 8), (19, 2), (28, 19), (12, 1), (16, 15), (12, 14), (22, 20), 
numberIntersection=0 
minDistance=3.794733192202055 
minAngle=0.08673833867598546 
minEdgeLength=5.0 
sumAngle=40.04250702580588 
sumEdgeLength=1424.6462239647256 
sumDistance=48796.63696079683 

0.0000000000 3.7947331922 0.0867383387 5.0000000000 40.0425070258 1424.6462239647 48796.6369607968
0: (39, 89),
1: (96, 44),
2: (102, 83),
3: (0, 78),
4: (93, 91),
5: (96, 87),
6: (120, 9),
7: (105, 37),
8: (111, 51),
9: (116, 64),
10: (116, 0),
11: (92, 68),
12: (114, 38),
13: (82, 96),
14: (110, 44),
15: (95, 77),
16: (104, 72),
17: (94, 37),
18: (107, 66),
19: (120, 110),
20: (62, 72),
21: (104, 52),
22: (102, 59),
23: (95, 82),
24: (102, 90),
25: (93, 96),
26: (107, 78),
27: (99, 37),
28: (86, 106),
29: (105, 45),
39 89 96 44 102 83 0 78 93 91 96 87 120 9 105 37 111 51 116 64 116 0 92 68 114 38 82 96 110 44 95 77 104 72 94 37 107 66 120 110 62 72 104 52 102 59 95 82 102 90 93 96 107 78 99 37 86 106 105 45 
0: (39, 89), 1: (96, 44), 2: (102, 83), 3: (0, 78), 4: (93, 91), 5: (96, 87), 6: (120, 9), 7: (105, 37), 8: (111, 51), 9: (116, 64), 10: (116, 0), 11: (92, 68), 12: (114, 38), 13: (82, 96), 14: (110, 44), 15: (95, 77), 16: (104, 72), 17: (94, 37), 18: (107, 66), 19: (120, 110), 20: (62, 72), 21: (104, 52), 22: (102, 59), 23: (95, 82), 24: (102, 90), 25: (93, 96), 26: (107, 78), 27: (99, 37), 28: (86, 106), 29: (105, 45), 
numberIntersection=0 minAngle=0.08673833867598546 sumAngle=40.0425070257723 minEdgeLength=5.0 sumEdgeLength=1424.64622396472 minDistance=3.794733192202055 sumDistance=48796.636960797056 
Time taken: 19.563 s
